pages/index.tsx

```diff
@@ -24,7 +24,7 @@ function Index() {
         The engine is still very limited in it's capabilities only passing about
         {" "}
         <a href={href(import.meta.resolve("./test262.tsx"))}>
-          33% of the test262 test suite
+          43% of the test262 test suite
```

Comment:
**suggestion:** Could probably be a dynamic variable we calculate from the latest commit like the graph generation does. Your choice if you want to do that :)

Reply:
maybe not for now

Reply:
next time we rework the graphs maybe though